docs/tutorials/scenario-simulation/planning-simulation/installation.md

```diff
@@ -0,0 +1,34 @@
+# Installation
+
+This document contains step-by-step instruction on how to build [AWF Autoware Core/Universe](https://github.com/autowarefoundation/autoware) with `scenario_simulator_v2`.
+
+## Prerequisites
+
+1. [Autoware has been built and installed](https://autowarefoundation.github.io/autoware-documentation/main/installation/)
+
+## How to build
+
+1. Navigate to the Autoware workspace:
+
+   ```bash
+   cd autoware
+   ```
+
+2. Import Simulator dependencies:
+
+   ```bash
+   vcs import src < simulator.repos
+   ```
+
+3. Install dependent ROS packages.
+
+   ```bash
+   source /opt/ros/galactic/setup.bash
+   rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO
+   ```
+
+4. Build the workspace.
+
+   ```bash
+   colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
+   ```
```

Comment:
[nits] The final characters are not consistent. Is that intended?

![image](https://user-images.githubusercontent.com/31987104/169840066-5c5dbb4a-1afe-4ded-b825-83c420e4d0f8.png)